### PR TITLE
[feature/RPA-601] Add aria-labelledby in sections.

### PR DESCRIPTION
**Pull request by star-in-the-jar**
I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -16,4 +16,8 @@
         "runbotics-common",
         "runbotics-cli",
     ],
+    "cSpell.words": [
```
Comment (star-in-the-jar) [inline .vscode/settings.json]: usunąć

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationContent/IntegrationContent.tsx [hunk 2]
```diff
@@ -3,14 +3,15 @@ import { FC } from 'react';
...
+import { INTEGRATIONS_TITLE_ID } from '../IntegrationSection.utils';
 import styles from './IntegrationContent.module.scss';
 
 const IntegrationSection: FC = () => {
     const { translate } = useTranslations();
 
     return (
         <div className={styles.root}>
-            <Typography className={styles.title}>
+            <Typography id={INTEGRATIONS_TITLE_ID} className={styles.title} variant="h2">
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationContent/IntegrationContent.tsx]: variant before classname

**Review by star-in-the-jar — COMMENTED** [review]
Self reviewed.

File: .vscode/settings.json (hunk 1 above)
Comment (star-in-the-jar) [reply .vscode/settings.json]: done

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationContent/IntegrationContent.tsx (hunk 2 above)
Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/IntegrationSections/IntegrationContent/IntegrationContent.tsx]: done

**Review by star-in-the-jar — COMMENTED** [review]
Self reviewed.

**Review by preludium — APPROVED** [review]

**Pull request merged**